dataland-frontend/src/components/resources/frameworkDataSearch/p2p/P2pPanel.vue

```diff
@@ -0,0 +1,332 @@
+<template>
+  <div v-if="waitingForData" class="d-center-div text-center px-7 py-4">
+    <p class="font-medium text-xl">Loading P2P Data...</p>
+    <em class="pi pi-spinner pi-spin" aria-hidden="true" style="z-index: 20; color: #e67f3f" />
+  </div>
+  <div v-if="mapOfKpiKeysToDataObjectsArrays.size > 0 && !waitingForData">
+    <DataTable tableClass="onlyHeaders">
+      <Column bodyClass="headers-bg" headerStyle="width: 30vw;" headerClass="horizontal-headers-size" header="KPIs">
```

Comment:
Lets use `<Column />` notation

Reply:
changed

Reply:
and not sure, but i have removed the bodyClass